src/data-sources/intermine/api/get-phylonode.js

```diff
@@ -17,5 +17,26 @@ async function getPhylonode(id) {
         });
 }
 
+// get a Phylonode for a Protein
+async function getPhylonode({protein}={}) {
```

Comment:
This function needs to return something when protein is not specified. As it currently is, when we switch to TypeScript the compiler will throw an error. Better yet, since protein is required, don't set the default to an empty object and remove the if block. This way an error will be thrown at run-time if someone tries to call the function without a protein argument.